dns/unboundbl/src/opnsense/scripts/OPNsense/Unboundbl/UnboundBL.sh

```diff
@@ -1,20 +1,35 @@
-# !/bin/sh
-# UnboundBL, UnboundBL.sh
+#!/bin/sh
+# The main worker script for UnboundBL.
+
+# init counter for debugging purposes
+cnt=0
+
+echo "Removing old files..."
+[ -f /var/unbound/dnsbl.conf ] && rm -f /var/unbound/dnsbl.conf
+[ -f /tmp/hosts.working ] && rm -f /tmp/hosts.working
 
 # include config-generated blacklist/whitelist, commas replaced with spaces
-source /usr/local/opnsense/scripts/OPNsense/Unboundbl/data.sh
+. /usr/local/opnsense/scripts/OPNsense/Unboundbl/data.sh
```

Comment:
I'm on the fence with this, "source" is more noticeable than "." but I use "." in my own stuff pretty often

Reply:
I ran into some strange errors while importing variables via “.”, while “source” never complained and acted predictably.